modules/eks/echo-server/main.tf

```diff
@@ -2,49 +2,32 @@ locals {
   enabled               = module.this.enabled
   ingress_nginx_enabled = var.ingress_type == "nginx" ? true : false
   ingress_alb_enabled   = var.ingress_type == "alb" ? true : false
-
-  alb_access_logs_enabled = var.alb_access_logs_enabled && var.alb_access_logs_s3_bucket_name != null && var.alb_access_logs_s3_bucket_name != ""
-  ingress_controller_group_enabled = var.enable_alb_controller_ingress_group ? [
```

Comment:
related to dd log forwarding test

Reply:
As above, Access Log settings are per Ingress group. We do not want `echo-server` being responsible for them. They should be set up via IngressClass in the `alb-controller` or other component.